teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1155 @@
+{
```

Comment:
<p>3 * MD Simulations -&gt; MD <strong>s</strong>imulations</p><p> </p><p>Not entirely sure what you mean with the last sentence: </p><blockquote>A more extensive introduction to applications of MD simulations in the context of drug design can be found in the following talktorial.</blockquote><p>An example of how to apply MD simulations in the context of drug design is provided within this talktorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I think it refers to the next talktorial, which analyzes the MD sim. It is confusing in this context and I removed it. The next notebook is referenced at the end </p>